.wordlist.txt

```diff
@@ -264,25 +298,32 @@ SSL
 startx
 StephanStS
 sudo
+symlink
 Syncthing
+syslog
 systemd
 TasmoAdmin
 Tasmota
 Tautulli
 TCP
+tcpdump
 TigerVNC
 Tonido
 txt
 uae
 UART
 Ubooquity
+UDP
 UEFI
 UI
 Unarchiver
+uncomment
+Unmount
 upgradable
 UPnP
 upvote
 UrBackup
+url
```

Comment:
Shall we use "url" or only "URL"? I used "URL" in my documents. 
What is better?

Reply:
Agree, let's consequently use capital letters. Also when I updated posts on the forum, I changed this to capital letters, was just to lazy to go through all threads once 😄.

Reply:
If @Petru aggrees, I will change this in the docs.

Reply:
Changed url -> URL